app/lib/features/chat/pages/profile_page.dart

```diff
@@ -180,16 +180,12 @@ class ProfilePage extends ConsumerWidget {
                         buildRequestsAndInvites(context),
```

Comment:
It seems like we still have widget methods residing in code structures, might take a look on that maybe separately, extracting them to widgets if possible to adhere to performance patterns.

Reply:
Okay, will fix them.